src/redux/reducers/compose.js

```diff
@@ -41,7 +41,8 @@ export default function(state = initialState, action) {
       return {
         ...state,
         hiddenAddAnswer: false,
-        answerInputText: ''
+        answerInputText: '',
+        answers: [...state.answers, action.payload]
```

Comment:
very nice spread.. so much better than that arr.push

Reply:
thanks!